test/annotation_layer_test.css

```diff
@@ -75,6 +77,26 @@
   padding-right: 0;
 }
 
+.annotationLayer .buttonWidgetAnnotation.checkBox label,
+.annotationLayer .buttonWidgetAnnotation.radioButton label {
```

Comment:
Unless I'm mistaken, most of these rules are the same as those used above in https://github.com/mozilla/pdf.js/pull/7898/files#diff-7d2639c28c78de0283c24b2c5d56eb60R46, hence moving these lines there (with possible necessary overrides here) might make the code more readable/maintainable.